test/lib/ansible_test/_internal/docker_util.py

```diff
@@ -276,6 +280,8 @@ def docker_run(args, image, options, cmd=None, create_only=False):
         # Only when the network is not the default bridge network.
         options.extend(['--network', network])
 
+    options.extend(['--ulimit', 'nofile=%s:%s' % (MAX_NUM_OPEN_FILES, MAX_NUM_OPEN_FILES)])
```

Comment:
The docs state that the hard limit is derived from the soft limit, so we should be able to simplify this:

```suggestion
    options.extend(['--ulimit', 'nofile=%s' % MAX_NUM_OPEN_FILES])
```

Reply:
Committed, pulled and checked locally - it works:)